src/apps/reservation-list/modal/reservation-details/digital-list-details.tsx

```diff
@@ -23,28 +23,26 @@ const DigitalListDetails: FC<DigitalListDetailsProps & MaterialProps> = ({
       {expiryDate && state === "readyForPickup" && (
         <ListDetails
           icon={ReservationsIcon}
-          title={t("reservationDetailsExpiresTitelText")}
-          // todo string interpolation
-          labels={`${t("reservationDetailsExpiresLabelText")} ${formatDate(
-            expiryDate
-          )}`}
+          title={t("reservationDetailsStatusTitleText")}
+          labels={t("reservationDetailsExpiresText", {
+            placeholders: { "@count": formatDate(expiryDate) }
```

Comment:
I don't get this: @count pointing at a date?

Reply:
Do you have an explanation?

Reply:
I fixed it